src/bin/ion/commands/generate/model.rs

```diff
@@ -106,19 +108,7 @@ impl From<FullyQualifiedTypeName> for FullyQualifiedTypeReference {
 
 impl Display for FullyQualifiedTypeReference {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        if self.parameters.is_empty() {
-            return write!(f, "{}", self.type_name.join("."));
-        }
-        write!(f, "{}<", self.type_name.join("."))?;
-
-        for (i, parameter) in self.parameters.iter().enumerate() {
-            if i == self.parameters.len() - 1 {
-                write!(f, "{}", parameter)?;
-            } else {
-                write!(f, "{},", parameter)?;
-            }
-        }
-        write!(f, ">")
+        write!(f, "{}", self.string_representation::<JavaLanguage>())
```

Comment:
This stands out as being a Java-specific method call in a language-agnostic method. Are you using Java FQTR syntax as the abstract syntax too? If so, please add an explanatory comment. If not, how does this behave when generating Rust?

Reply:
This `Display` is not being used now. It will only use `string_representation` method on `FulylQualifiedTypeReference` which takes a parameter `L: Language` to specify which programming language to be represented with this.